### DISCUSSION TITLE
Best setup for optimal performance

### DISCUSSION THREAD

**Discussion by bsjdsp** [apohl79/audiogridder#445] [Q&A]
Is there a special cable to use or gigabit network and other computer related settings to achieve the best settings and performance.

**Comment by CaBuffalo** [comment]
Gigabyte network is more than enough.

On Mon, Mar 22, 2021 at 2:18 PM bsjdsp ***@***.***> wrote:

> Is there a special cable to use or gigabit network and other computer
> related settings to achieve the best settings and performance.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/445>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6AVSOPUGQJJW5V6GSTTE6XZ3ANCNFSM4ZT45QWA>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by bsjdsp** [reply]
So then I guess 10 gigabit network would be overkill I guess.

**Comment by timbit2006** [comment]
I found better results setting up a static IP address on my slaves. on my master computer the IP is dynamic

**Comment by CaBuffalo** [comment]
Good to know. Thanks for the feedback.

On Mon, Mar 22, 2021 at 11:33 PM timbit2006 ***@***.***>
wrote:

> I found better results setting up a static IP address on my slaves. on my
> master computer the IP is dynamic
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/445#discussioncomment-516390>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6B47Z763SK433RBQC3TFAY2RANCNFSM4ZT45QWA>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Comment by CaBuffalo** [comment]
It's good to have but I think it is overkilled. :)

On Wed, Mar 24, 2021 at 1:38 AM bsjdsp ***@***.***> wrote:

> So then I guess 10 gigabit network would be overkill I guess.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/445#discussioncomment-521704>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6E3EZGSOZO6UGPZJ7LTFGQIJANCNFSM4ZT45QWA>
> .
>


-- 
Kevin Ma
Supported by Gmail.
